src/components/WalletBox/WalletBox.js

```diff
@@ -28,7 +28,10 @@ const formatAssets = balances => {
 	const { eth, snx, synths } = balances;
 	let assets = [{ name: 'ETH', ...eth }];
```

Comment:
would be great to use `CRYPTO_CURRENCY.ETH` for `ETH` string

Reply:
`CRYPTO_CURRENCY_MAP.ETH` was the correct one.